src/main/java/org/fundacionjala/pivotal/pages/Project.java

```diff
@@ -0,0 +1,79 @@
+package org.fundacionjala.pivotal.pages;
+
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.springframework.stereotype.Component;
+
+import org.fundacionjala.core.ui.AbstractPage;
+
+/**
+ * this class represented a login page.
+ */
+@Component
+public class Project extends AbstractPage {
+
+    @FindBy(id = "create-project-button")
+    private WebElement createNewProjectButton;
+
+    @FindBy(css = ".tc-form__input")
+    private WebElement projectNameField;
+
+    @FindBy(css = ".tc-account-selector")
+    private WebElement accountSelector;
+
+    @FindBy(css = ".tc-account-selector__option-list li:nth-of-type(1)")
+    private WebElement accountSelectorOptionone;
+
+    @FindBy(xpath = "//*[@id=\"modal_area\"]/div/div[2]/div/form/footer/button[2]")
```

Comment:
@AWT02/awt02 
this is a wrong locator
We will review this in the class

Reply:
locator has changed